src/compas_fab/sensors/baumer.py

```diff
@@ -8,8 +8,7 @@
 
 
 class PosCon3D(SerialSensor):
-    """Provides an interface for the `Baumer PosCon3D edge measurement sensor
-    <http://www.baumer.com/us-en/products/distance-measurement/light-section-sensors/poscon-3d-edge-measurement/>`_.
+    """Provides an interface for the Baumer PosCon3D edge measurement sensor.
```

Comment:
I searched the Baumer website a bit looking for replacement links, but I didn't find anything suitable.